internal/yggdrasil/yggdrasil.go

```diff
@@ -392,6 +404,39 @@ func (h *Handler) updateDeviceStatus(ctx context.Context, device *v1alpha1.EdgeD
 	return err
 }
 
+func (h *Handler) updateDeviceLabels(ctx context.Context, device *v1alpha1.EdgeDevice, labels map[string]string) error {
```

Comment:
WDYT about moving this code to the EdgeDevice repository?

Reply:
sure, will move